scripts/run-tsbs.sh

```diff
@@ -43,22 +68,33 @@ go build ./cmd/tsbs_load_ceresdb
 go build ./cmd/tsbs_generate_queries
 go build ./cmd/tsbs_run_queries_ceresdb
 
-# generate benchmark data
-./tsbs_generate_data --use-case="cpu-only" --seed=123 --initial-scale=${HOST_NUM} --scale=${HOST_NUM} \
-                     --timestamp-start="${TS_START}" \
-                     --timestamp-end="${TS_END}" \
-                     --log-interval="60s" --format="${FORMATS}" > ${DATA_FILE}
+if [ ! -f ${DATA_FILE} ]; then
+  # Generate benchmark data if it does not exist.
+  ./tsbs_generate_data \
+    --use-case="cpu-only" \
+    --seed=123 \
+    --initial-scale=${HOST_NUM} \
+    --scale=${HOST_NUM} \
+    --timestamp-start="${TS_START}" \
+    --timestamp-end="${TS_END}" \
+    --log-interval="60s" \
+    --format="${FORMATS}" > ${DATA_FILE}
+fi
 
 
-# write data to ceresdb
-./tsbs_load_ceresdb --file ${DATA_FILE} | tee ${LOG_DIR}/write.log
+# Write data to ceresdb
+./tsbs_load_ceresdb --ceresdb-addr=${CERESDB_ADDR} --file ${DATA_FILE} | tee ${LOG_DIR}/write.log
+    
 
-# generate queries
+# Generate queries for query
 ./scripts/generate_queries.sh
 
-# run queries against ceresdb
-# Note: only run 5-8-1 now
-cat ${BULK_DATA_DIR}/ceresdb-single-groupby-5-8-1-queries.gz | gunzip | ./tsbs_run_queries_ceresdb | tee ${LOG_DIR}/5-8-1.log
+# Run queries against ceresdb
+# TODO: support more kinds of queries besides 5-8-1.
+cat ${BULK_DATA_DIR}/ceresdb-single-groupby-5-8-1-queries.gz | gunzip | ./tsbs_run_queries_ceresdb --ceresdb-addr=${CERESDB_ADDR} | tee ${LOG_DIR}/5-8-1.log
+
+# Clean the result file
+rm -rf ${RESULT_FILE}
```

Comment:
It seems `-rf` is not necessary, and this is very dangerous. maybe we can remove it?